sdk/python/feast/job_service.py

```diff
@@ -44,8 +53,8 @@
 
 
 class JobServiceServicer(JobService_pb2_grpc.JobServiceServicer):
-    def __init__(self):
-        self.client = feast.Client()
+    def __init__(self, client):
+        self.client = client
 
     def _job_to_proto(self, spark_job: SparkJob) -> JobProto:
```

Comment:
Question: Why is `_job_to_proto` a method and not a function?

Reply:
That wasn't written by me, but I can change it if necessary. @oavdeev any thoughts / reasons not to change this?

p.s. I don't like this becoming a public function on SparkJob because the users will see it, and it's not a publicly facing functionality. I think a private function on SparkJob would be better.

Reply:
Yeah it could be a standalone function too. I didn't want to put it on `SparkJob` to keep it a pure interface. Besides, it is only used by JobServiceServicer.

Reply:
> I didn't want to put it on SparkJob to keep it a pure interface

Wouldn't it still be a pure interface? I mean we don't have to implement `to_proto()` on the ABC, but only the children.

Reply:
> I dont think it should be a method.
What do you mean? If it's not a method, you mean it should be a standalone function? Or do you mean that it shouldn't be a method of JobServiceServicer class specifically?